standards/root-cause-analysis.md

```diff
@@ -6,9 +6,11 @@ The root cause analysis SHOULD be done as soon as possible _after_ the incident
 
 RCAs are *blameless*. We're all human and make mistakes. At NYPL we want to encourage an [engineering culture](../culture/values.md#not-shaming-or-blaming) of experimentation, learning, continuous improvement, and making room for mistakes.
 
-Completed RCAs SHOULD be saved in Confluence, under the space for your project. This is a directory for a historical record and shared with relevant parties.
```

Comment:
As far as I can tell this isn't being followed, and generally we are taking another approach with TADs, so perhaps we should recommend a different method here? Right now I think I'm okay with this being vague

Reply:
Are RCAs currently stored in drive?  What happened to the two that we did?  In general I agree - let's get the TAD stuff finalized first.

Reply:
They were sent to Garvita, and she said that they should probably go to Jessica as well, but we didn't discuss storing them in a shared drive or anything like that. It's probably deserving of it's own conversation then we can add more detail here once we have a real decision.